library/photon

```diff
@@ -4,24 +4,24 @@ Maintainers: Fabio Rapposelli <fabio@vmware.com> (@frapposelli),
 GitRepo: https://github.com/vmware/photon-docker-image.git
 Directory: docker
 
-Tags: 4.0, 4.0-20220617, latest
+Tags: 4.0, 4.0-20220624, latest
 Architectures: amd64, arm64v8
-GitFetch: refs/heads/x86_64/4.0-20220617
-GitCommit: 26dc28cc9e3750ac7a782c47698ecef6e373969d
-arm64v8-GitFetch: refs/heads/aarch64/4.0-20220617
-arm64v8-GitCommit: 40b034a4b6fdcb7ab3401e4512795e07f3288bdd
+GitFetch: refs/heads/x86_64/4.0-20220624
+GitCommit: c8f1c629772472f8e9dddda0b9499d009ee545fb
+arm64v8-GitFetch: refs/heads/aarch64/4.0-20220624
+arm64v8-GitCommit: 466b6715f7f5bdf0d5f76ab1d9a5628b63539097
 
-Tags: 3.0, 3.0-20220617
+Tags: 3.0, 3.0-20220624
 Architectures: amd64, arm64v8
-GitFetch: refs/heads/x86_64/3.0-20220617
-GitCommit: dd8799ae99fc065c02ac8f2fb397f0d7d85b12c9
-arm64v8-GitFetch: refs/heads/aarch64/3.0-20220617
-arm64v8-GitCommit: bb3d18fc3de752b730cc0e8cf0f86ca329809d9e
+GitFetch: refs/heads/x86_64/3.0-20220624
+GitCommit: 41f22b9aadc0fc9be4ec32b60b961505b80e7005
+arm64v8-GitFetch: refs/heads/aarch64/3.0-20220624
+arm64v8-GitCommit: ea8eea96c94736f8c33c7783ebcc4f5b02cd866e
 
 Tags: 1.0, 1.0-20220429
```

Comment:
This tag should probably be removed now, right? (https://blogs.vmware.com/vsphere/2022/01/photon-1-x-end-of-support-announcement.html)

Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Hi @tianon thanks for bringing our attention to this. We will remove 1.0 tag with the followup update.

Reply:
Nice, looking forward to it! :smile:

(I'll merge this for now :+1:)